app/flows/marriage_abroad_flow/outcomes/countries/vietnam/uk/partner_other/_same_sex.erb

```diff
@@ -55,10 +55,9 @@ Receiving a notice of registration | <%= format_money calculator.consular_fee(:r
 Registering a marriage | <%= format_money calculator.consular_fee(:registering_marriage) %>
 Issuing a marriage certificate | <%= format_money calculator.consular_fee(:issuing_civil_partnership_or_marriage_certificate) %>
 
-You can pay fees for consular services in the local currency or pounds sterling - these are shown in the list of [consular fees for Vietnam](/government/publications/vietnam-consular-fees).
+Fees are shown in the list of [consular fees for Vietnam](/government/publications/vietnam-consular-fees). 
 
-<%=  render partial: 'how_to_pay',
-            locals: {calculator: calculator}%>
+It's quicker and easier to pay by card (Visa or Mastercard). You can also pay in cash if you pay in the local currency.
```

Comment:
I noticed that this is missing the link that is applied in `vietnam/_opposite_sex_civil_partnership.erb` (https://github.com/alphagov/smart-answers/blob/093bd82d488a2a9deea789dd02bcfcf85b9dce27/app/flows/marriage_abroad_flow/outcomes/countries/vietnam/_opposite_sex_civil_partnership.erb#L21). For consistency, do we want the link in all outcomes that use this text?

```suggestion
It's quicker and easier to pay by card (Visa or Mastercard). You can also pay in cash if you pay in the [local currency](/government/publications/vietnam-consular-fees).
```

Reply:
Hi Chris, thanks for checking and for spotting this! If we could add that link in, that would be great, thanks. Would you mind if we keep the self-contained outcomes approach for now, please? Many thanks